docs/2.deploy/20.providers/firebase.md

```diff
@@ -1,26 +1,53 @@
 # Firebase
 
-> Deploy Nitro apps to Firebase.
+> Deploy Nitro apps to Firebase hosting.
 
-**Preset:** `firebase`
+::note
+You will need to be on the [**Blaze plan**](https://firebase.google.com/pricing) (Pay as you go) to get started.
+::
 
-:read-more{title="Firebase Hosting" to="https://firebase.google.com/docs/hosting"}
+## Firebase app hosting <sup>(beta)</sup>
```

Comment:
Could we separate out the doc changes from the code changes into different prs? We'd probably want to get the code changes merged asap but might hold off slightly on the doc changes until we have our docs + console updated fully to reflect that we also support other frameworks outside of Angular + Nextjs

Reply:
Surely i can hide it for time being